packages/runtime-core/src/componentOptions.ts

```diff
@@ -753,6 +754,8 @@ export function applyOptions(
     if (__COMPAT__ && isCompatEnabled(DeprecationTypes.FILTERS, instance)) {
       resolveInstanceAssets(instance, options, FILTERS)
     }
+
+    if (inheritAttrs !== undefined) instance.type.inheritAttrs = inheritAttrs
```

Comment:
This is not ideal - we should never mutate `instance.type` based on the mixin it is using.

I think we need to add a field on `ComponentInternalInstance` instead.

Reply:
make sence.
DONE~